backend/Controllers/AuthenticationController.cs

```diff
@@ -0,0 +1,35 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+using CbtBackend.Models;
+using CbtBackend.Entities;
+using CbtBackend.Services;
+
+namespace CbtBackend.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class AuthenticationController : ControllerBase {
+    private readonly ILogger<AuthenticationController> logger;
+    private readonly IUserService userService;
+
+    public AuthenticationController(IUserService userService, ILogger<AuthenticationController> logger) {
+        this.logger = logger;
+        this.userService = userService;
+    }
+
+    [AllowAnonymous]
+    [HttpPost(Name = "PostAuthentication")]
+    public IActionResult Post(UserAuthenticationRequest authenticationRequest) {
+        logger.LogInformation("authenticating user with data [email = {email}, password = {password}]",
+            authenticationRequest.Email, authenticationRequest.Password);
```

Comment:
we definitely do not want to log passwords :eyes:

Reply:
it is now logged only during debugging, changed in https://github.com/shilangyu/SE2-CBT/pull/124/commits/1fc395316fa969ddc58ee8d17d9cdef9fa900d01